searx/templates/simple/answer/weather.html

```diff
@@ -0,0 +1,63 @@
+{% macro show_weather_data(answer, data) %}
+  <table>
+    <colgroup>
+      <col span="1" class="thumbnail">
+      <col span="1" class="title">
+      <col span="1" class="measured">
+      <col span="1" class="title">
+      <col span="1" class="measured">
+      <col>
+    </colgroup>
+    <tbody>
+      <tr>
+        <td rowspan="4">
+          {%- if data.url %}<img class="symbol" src="{{ data.url }}" title="{{ data.summary }}">{% endif -%}
```

Comment:
I think this should be in the bottom right of the result like for the other answerers and not in the summary line. Also, shouldn't this point you to the web ui of the weather engine with the location already entered instead of just to their docs? I think it's not really helpful for the user this way.